fbchat/graphql.py

```diff
@@ -214,7 +214,7 @@ def graphql_to_user(user):
     if user.get('profile_picture') is None:
         user['profile_picture'] = {}
     c_info = get_customization_info(user)
-    plan = graphql_to_plan(user['event_reminders']['nodes'][0]) if user.get('event_reminders') else None
+    plan = graphql_to_plan(user['event_reminders']['nodes'][0]) if user.get('event_reminders', dict()).get('nodes') else None
```

Comment:
Be aware that `user.get('event_reminders', dict()).get('nodes')` can still raise an `AttributeError`, on data that looks like e.g:
```py
user = {
    'event_reminders': None
}
```
But it might not be necessary to check this case

Reply:
I've never seen `None` in `event_reminders`, but maybe I'm wrong.